metabase/metabase
# Backport dashboard performance fixes

### Description

Do not autoapprove since it is not a cherry-pick but a manual transfer of the changes.

Manual backport of the following:
- https://github.com/metabase/metabase/pull/31963
- https://github.com/metabase/metabase/pull/31610 — only dashboard stickiness changes from here
- https://github.com/metabase/metabase/pull/32245

### How to verify

Create a dashboard with filters, ensure the sticky filter work in all contexts including the full app embedding
